app/models/envelop.rb

```diff
@@ -0,0 +1,8 @@
+class Envelop < ApplicationRecord
```

Comment:
@atatarenko, I would argue here. I don't expect this to be useless code. 

First of all, I did create it because I saw the future extension to categories. But "category" sounds too abstract to me. 

The second reason is that, in the current module, I have a task to display the current balance of a user in the header. Thus I have two options here: 1) calculate it each time based on each transaction he made, 2) generate a separate model to store and update this data upon each transaction occurred. 

Now imagine that user has 100k transactions. How long would balance calculate based on option #1 while making 100k SQL calls to DB? With option #2 we would have only one call to respective table. 

At the same time, we can treat this 'balance' instance as the 'unallocated' category from module 4. 

I know everything has its time, but planing the architecture ahead never killed anyone, didn't it?